Packs/Core/TestPlaybooks/Test_Playbook_-_Identity_Analytics_-_Alert_Handling.yml

```diff
@@ -0,0 +1,2071 @@
+id: Test Playbook - Identity Analytics - Alert Handling
```

Comment:
We should test the playbook twice: once with the Azure user flow and once with the Okta user flow, using our dedicated test users for each. Additionally, to verify the remediation actions, let's run the playbook with auto-remediation and auto-containment inputs set to 'true'.

Reply:
done - added more tasks to check Okta user flow as well.
Regarding the remediation and containment - I asked Yarden and she told me there is no need to test sub-playbooks.